mattermost-plugin/webapp/src/components/rhsChannelBoards.tsx

```diff
@@ -78,6 +84,11 @@ const RHSChannelBoards = () => {
     if (!currentChannel) {
         return null
     }
+
+    if (!dataLoaded) {
```

Comment:
this prevents the component to blink when changing from zero state to listing the boards state.